### PR TITLE
Free as the wind (Map for MT CTF)

### PR DESCRIPTION
**Pull request by EISHU92**
This is a flat map, but with giant space to use.

### CODE REVIEW

**Comment by olliy1or** [comment]
screenshot is missing, take a nice screenshot of the whole map(with chat and huds off) and upload it there as `screenshot.png`

**Comment by EISHU92** [comment]
Ok

**Comment by EISHU92** [comment]
I maked the screenshot.png now it still there

**Comment by EISHU92** [comment]
If it continues with some problems, dont worry, ill solve it

**Comment by EISHU92** [comment]
If you want, i can do a video of a game play

**Comment by EISHU92** [comment]
I changed screenshot of that map.

**Comment by HobbitPower** [comment]
* I guess the screenshot should have a ratio of 3:2.  (also parts of the barrier is not loaded there).
* Chestzones, team zones, outer barriers, red barrier are working fine so far, found no bug there yet. 

* only thing which was wrong with the barrier is, that there is no top layer, so your map is open to the sky. don't know how important it is to have one, because probably you can't get out there anyway.
* I don't really understand why you are able to dig down at the "grass/dirt field" at the red side, but you can't do it at the blue side. You are not able to tunnel there/get to the blue side, since there are unbreakable blocks everywhere. The only advantage would be that the red side could mine more iron/coal. Might wanna fix that
* There is a small spelling mistake in your conf file (look at the hint line)

I am not really a fan of the design of the map at all. Too flat & too open. A paradise for pillarers, block spammers and sharpshooters/scouts. Probably thats the intention behind the map, but personally i wont play there (often).

**Comment by EISHU92** [comment]
oh ok

File: freedomap/map.conf [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+phys_jump = 2
+phys_speed = 2
+size = return {["z"] = 220, ["x"] = 220, ["y"] = 140}
+map_version = 2
+enabled = true
+name = Freely as the wind
+phys_gravity = 1
+initial_stuff = return {"default:pick_stone", "ctf_ranged:pistol_loaded", "default:torch 5"}
+author = KareaGhandTrail
+hint = Get everyting before your enemy gets!
```
Comment (MoreKat) [inline freedomap/map.conf:14]: This hint is a) grammatically incorrect (should be 'Get **everything** before your enemy gets **it**', and b) useless

**Comment by LuismiInTheZone** [comment]
> @EISHU92 Could you let me know if you still want to work on the map in the future?

https://discord.com/channels/447819017391046687/448136831385468930/1098366238704537670 He said yes

**Comment by HobbitPower** [comment]
I saw it Luis, thx. I'll still give them some time to think about it. 
But if they are not want to change anything about the map, even not the suggested changes, I will disapprove the map anyway. Its way too plain and open.

**Comment by HobbitPower** [comment]
I am closing the PR as there is no more interest in the map and there will be no changes. And in its current form, it is not sufficient for a merge. They were asked if they would like to continue working on it + were informed about the decision here, via PM and in [#maps](https://discord.com/channels/447819017391046687/448136831385468930/1098366201056481363), where they also agreed.

**Pull request closed without merging**